### PR TITLE
Refactor rank page url

### PR DESCRIPTION
**Pull request by SidharthK2**
# Refactor rank page URL to be more shareable

New rank page URL :  `/rank/[category]`

image

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/1976
closes https://github.com/EveripediaNetwork/issues/issues/1996

### CODE REVIEW

File: src/pages/rank/[id].tsx [hunk 1]
```diff
@@ -89,8 +89,12 @@ const Rank = ({
   const [nftItems, setNftItems] = useState<RankCardType[]>([])
   const [founderItems, setFounderItems] = useState<RankCardType[]>([])
   const [sortOrder, setOrder] = useState<SortOrder>('descending')
+
   const router = useRouter()
-  const { pathname } = router
+  const { p: page, id } = router.query
+  //TODO: remove console.log
+  console.log('id', id)
+  console.log('pg', page)
```
Comment (Aliiiu) [inline src/pages/rank/[id].tsx:97]: You can clean this

File: src/components/Rank/RankPagination.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ const RankPagination = (props: RankpaginationProps) => {
       pathname,
       query: {
         ...query,
-        page: pageNumber,
+        p: pageNumber,
```
Comment (Aliiiu) [inline src/components/Rank/RankPagination.tsx:49]: I think page is better

**Review by Aliiiu — CHANGES_REQUESTED** [review]

File: src/pages/rank/[id].tsx (hunk 1 above)
Comment (Adebesin-Cell) [reply src/pages/rank/[id].tsx:97]: Not a big deal, I think it’ll read better with [caregory].tsx 👍.

Comment (SidharthK2) [reply src/pages/rank/[id].tsx:97]: @Adebesin-Cell Agreed.

**Review by Aliiiu — APPROVED** [review]

**Pull request merged**